CryptoPkg/Driver/Packaging/generate_cryptodriver.py

```diff
@@ -689,9 +689,9 @@ class options():
                 for arch in arches:
                     if arch == "ARM":
                         continue
-                    if arch == "AARCH64" and phase != "Dxe":
+                    if (arch == "ARM" or arch == "AARCH64") and phase == "Smm":

```

Comment:
I agree the file has style issues. However, that would best be addressed in a rewrite of the file (if we cared).

I'll update these since the lines are being modified.